tests/QueryTest.php

```diff
@@ -119,13 +113,13 @@ function testQueryShouldEncodeUrlParameters() {
         $q = $this->client->query();
         $q->pattern = 'spotify';
         $q->end_time = \DateTime::createFromFormat('Y-m-d H:i:s', '2012-12-28 09:01:22');
-        $this->assertContains('tsTo=2012-12-28+09%3A01%3A22', $q->url_parameters());
+        $this->assertContains('end-date%3A2012-12-28T09%3A01%3A22', $q->url_parameters());
     }
 
     function testQueryPattern() {
         $q = $this->client->query();
         $q->pattern = 'spotify';
-        $this->assertContains('searchpattern=spotify', $q->url_parameters());
+        $this->assertContains('spotify', $q->url_parameters());
     }
 
     public function testExecuteQueryWithInvalidAPIKey() {
```

Comment:
This is the test that fails on Travis. It just has the wrong name (I believe is's just an old copy-paste error). If you read what the test does, it makes sure a valid request returns posts. You can rename it to `testExecuteWithValidQuery`, since that is what the test actually does.

The cassette for that test also needs to be re-recorded with a valid API key.

**Make sure you don't commit the API key though.** It looks like it can be filtered out using the solution in https://github.com/php-vcr/php-vcr/issues/174#issuecomment-260558738.
I haven't tested whether the filter works though. If it doesn't work, just edit the cassette by hand before committing and we'll open an issue about fixing it. I recently discovered that we have the same problem in our Nodejs client as well.